Tests/tinys3Tests/model/AWSCredentialsTests.swift

```diff
@@ -3,13 +3,80 @@ import XCTest
 
 final class AWSCredentialsTests: XCTestCase {
 
-    let credentials: AWSCredentials = .testDefault
+    func testThatSingleFileContainsValidCredentials() throws {
+        let profiles = try AWSProfileConfig.profiles(from: R.AWSCredentialsFile.single, isCredentialsFile: true)
+        let defaultConfig = try XCTUnwrap(profiles["default"])
```

Comment:
Nit – `default` is super common – can we have a type here to accept that?

Reply:
Addressed in ec82c84f1d92c221123b8c17a3d5af6cb88cdaa2